docs/index.bs

```diff
@@ -3007,6 +3017,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
           The |task| *must* use |activeWorker|'s <a>event loop</a> and the <a>handle fetch task source</a>.
 
       1. Wait for |task| to have executed or for |handleFetchFailed| to be true.
+      1. If |handleFetchFailed| is false and |eventCanceled| is false, then resolve |eventHandled| with undefined, otherwise reject it with "{{NetworkError}}" {{DOMException}}.
```

Comment:
Use [=resolve=] and [=reject=]. You don't need to say "with undefined", that's the default.

"reject it with a" rather than just "reject it with". And again I'd specify the realm.

I'm not sure the condition is right here. It's only an exception if |respondWithEntered| is false and |eventCanceled| is true.

It might be better to handle resolving before each return in the next few lines, then we aren't duplicating the logic and risking them getting out of sync.

Reply:
Done